source/boards/accessing-boards.rst

```diff
@@ -0,0 +1,12 @@
+Accessing Focalboard from Mattermost
+=====================================
+
+Focalboard currently uses channel-based permissions, meaning that only members of the associated channel can access (read/write) the boards for that channel.
+
+You can use this to create private boards:
+
+1. Create or join a Private channel, Group Message, or Direct Message.
+2. Select the **Focalboard Workspace** icon in the channel header.
```

Comment:
Is it safe to assume that this icon name is also changing to Boards or Boards Workspace?

Reply:
Great question. Right now it's in flux, kind of like how Playbooks was, and it says "Focalboard Workspace" on community. This will no doubt change but I am leaving it as-is for now.